examples/asr/speech_multitask/speech_to_text_aed.py

```diff
@@ -66,6 +66,14 @@ def main(cfg):
 
     trainer = pl.Trainer(**cfg.trainer)
     exp_manager(trainer, cfg.get("exp_manager", None))
+
+    # Check for canary spl tokens to create spl_tokenizer.
+    if cfg.get("spl_tokens"):
+        logging.info("Detected spl_tokens config. Building tokenizer.")
+        spl_cfg = cfg["spl_tokens"]
```

Comment:
This is my bad, I should have been cleared about steps.

Now we're duplicating the class in both here and in the model. Do it one place, inside the model.

Read the target class from Cfg.model.tokenizer.custom_tokenizer
Load the class from class path using NeMo.utils.model_utils.import_class_from_path
Then call it's static method to build the tokenizer, passing in the kwargs of spl_tokens config dict

Reply:
Oh, that's what you mean. Gotcha.